### ISSUE TITLE
Amethyst Focus Follows Mouse Not Working With Fullscreen Windows

### ISSUE THREAD

**Issue by s26haide** [ianyh/Amethyst#798]
## System

 - **Amethyst version**: 0.12.2
 - **macOS version**: 10.14.2

## What's the problem?
With any window in fullscreen on 1 monitor, restarting amethyst makes focus follow mouse not work with the fullscreen window. However, if you un-fullscreen the window, it will work. Once it works like that, fullscreening the window again will fix the problem (but it's annoying to un-fullscreen and fullscreen my windows on amethyst restart) 

## How can it be reproduced?
Open a fullscreen window on 1 monitor, restart amethyst and observe that focus follows mouse does not work when going to the fullscreen window. Other non-fullscreen windows will focus properly.

## What applications are involved?
I've tested XCode, slack, and ios simulator but I'm pretty sure it happens for all apps

## Has anything fixed it, even temporarily?
Un-fullscreen the window and fullscreening again will make it work but it's annoying to do every time

## dotfile (if you have one)
N/A

## Screenshots
N/A

**Comment by ianyh** [comment]
We totally ignore windows in fullscreen spaces. Focus following mouse in a multi-display environment is an interesting case that I had not considered.

**Comment by s26haide** [comment]
For me personally, when I use multiple displays I always end up typing in the wrong place so I was looking for something like focus follow mouse. I think they go well together (would definitely improve my workflow) so do you think this is something that can be implemented in the future?

**Comment by ianyh** [comment]
Yeah. It makes sense to ignore the windows for the purposes of tiling because that doesn't really make sense when the window is in the native fullscreen environment. It seems pretty reasonable to allow focus changes, though.

**Comment by Catbodi** [comment]
> Yeah. It makes sense to ignore the windows for the purposes of tiling because that doesn't really make sense when the window is in the native fullscreen environment. It seems pretty reasonable to allow focus changes, though.

Thank you for clarifying I also had this issue.

I'm running 0.12.3b on 10.14.3. Specifically Google Chrome now works with Focus Follows Mouse on my mac however Messages does not work. I've tested numerous numerous other apps and they all work flawlessly except for messages. Any way to resolve?

**Comment by ianyh** [comment]
Sounds like #226 wasn’t actually fixed, but that’s a separate issue. I’ve reopened it.
